scripts/delete-task.js

```diff
@@ -1,13 +1,13 @@
 import { CLICK_EVENT } from "/scripts/constants.js";
 
 function onDeleteTask(deleteButton, task, taskListElement, tasks, id) {
-  deleteButton.addEventListener(CLICK_EVENT, deleteTask);
-
   function deleteTask() {
     taskListElement.removeChild(task);
     delete tasks[id];
     deleteButton.removeEventListener(CLICK_EVENT, deleteTask);
   }
+
+  deleteButton.addEventListener(CLICK_EVENT, deleteTask);
```

Comment:
what difference does it make to move the 'click` event registration to the bottom of the function?

Reply:
Doesn't make any difference.